src/main/java/transaction/TransactionList.java

```diff
@@ -31,33 +30,87 @@ public static void printAllTransactions() {
         System.out.println("Here are all the transactions: ");
 
         for (Transaction transaction : transactionList) {
-            System.out.println(index + ") " + transaction.getTransactionId()
-                    + " | " + transaction.getCarLicensePlate()
-                    + " | " + transaction.getBorrowerName()
-                    + " | " + transaction.getDuration()
-                    + " | " + transaction.getStartDate());
+            System.out.println(index + ") " + transaction);
```

Comment:
Hmmmm..... Does this actually work? Maybe we could revert back to the previous iteration so that we can specify our own format??

Reply:
pass an object to println is same as pass object.tostring()